torchrl/objectives/value/advantages.py

```diff
@@ -597,7 +619,7 @@ def value_estimate(
 
         if self.average_rewards:
             reward = reward - reward.mean()
-            reward = reward / reward.std().clamp_min(1e-4)
+            reward = reward / reward.std().clamp_min(1e-5)
```

Comment:
is there a reason to change this?
These small changes can introduce unforseen effects in our benchmarks. If we need to adapt it for the purpose of this PR I guess we should use an arg or smth like that to control it